src/tiffimage_int.hpp

```diff
@@ -155,17 +155,16 @@ using TiffGroupTable = std::unordered_map<TiffGroupKey, NewTiffCompFct, TiffGrou
          use standard TIFF layout.
 */
 struct TiffTreeStruct {
-  using Key = std::pair<uint32_t, IfdId>;
-  //! Comparison operator to compare a TiffTreeStruct with a TiffTreeStruct::Key
-  bool operator==(const Key& key) const;
+  const IfdId parentGroup_;      //!< Parent group
```

Comment:
why are these const?

Reply:
You're right: I didn't need to do that. I also didn't need to add the constructor. I've removed them.